### DISCUSSION TITLE
大佬方便吗，想使用autoduty

### DISCUSSION THREAD

**Discussion by jiujiuovo1** [DeepseaXX/aakaigua#6] [Q&A] [answered]
国服，安装了ad，atoman版的vnav、niguang大佬的bm、和rsr，但是点击run就会一动不动，难道是缘分不到吗

**Comment by DeepseaXX** [comment, marked as answer]
看下path中有可用的路线吗。
如果连不上github，可能无法顺利下载path配置。
可以自己去github下完放在插件对应目录里面。

**Reply by DeepseaXX** [reply]
顺便可以艾特我这样我可以更快看到
